src/platforms/ios/ioscontroller.swift

```diff
@@ -227,11 +204,22 @@ public class IOSControllerImpl : NSObject {
 
                 Logger.global?.log(message: "Loading the tunnel succeeded")
 
-                // If we were switching server, now it's time to consider the operation completed.
-                switchingServer = false
-
                 do {
-                    try (self.tunnel!.connection as? NETunnelProviderSession)?.startTunnel()
+                    if (reason == 1) {
```

Comment:
Maybe write a comment about the 'reason == 1'.

Reply:
Sure I will add one.